com.github.quaternion.json

```diff
@@ -0,0 +1,21 @@
+{
+    "id": "com.github.quaternion",
+    "branch": "master",
+    "rename-icon": "quaternion",
+    "rename-desktop-file": "quaternion.desktop",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "quaternion",
+    "tags": ["nightly"],
+    "desktop-file-name-prefix": "(Nightly) ",
+    "finish-args": ["--share=ipc", "--share=network", "--socket=x11", "--socket=wayland", "--env=QT_QPA_PLATFORM=flatpak", "--device=dri"],
+
+    "modules": [
+        {
+            "name": "quaternion",
+            "cmake": true,
+            "sources": [ { "type": "git", "url": "git://github.com/Fxrh/Quaternion.git" } ]
```

Comment:
Flathub needs reproducible builds. You need a specific commit to build.

Reply:
Further, this URL isn't in the https://github.com/QMatrixClient/Quaternion repo, is that correct?